docs/react-wiki-archive/Component-Implementation-Guide.md

```diff
@@ -0,0 +1,264 @@
+# Table of Contents
```

Comment:
Potential Keeper

Reply:
This also needs to be thoroughly scrubbed and updated with Martin if we chose to keep it.

Reply:
this should be re-worked/moved to v9 docs

Reply:
PR Buddy == Martin, CC @layershifter